project/src/pages/main-page/main-page.tsx

```diff
@@ -18,55 +68,38 @@ function MainPage({ offerCardCount, offers }: MainPageProps) {
         <h1 className="visually-hidden">Cities</h1>
         <div className="tabs">
           <section className="locations container">
-            <ul className="locations__list tabs__list">
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to="#">
-                  <span>Paris</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to="#">
-                  <span>Cologne</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to="#">
-                  <span>Brussels</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link
-                  className="locations__item-link tabs__item tabs__item--active"
-                  to="#"
-                >
-                  <span>Amsterdam</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to="#">
-                  <span>Hamburg</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to="#">
-                  <span>Dusseldorf</span>
-                </Link>
-              </li>
-            </ul>
+            <LocationsList activeCity={activeCity} />
           </section>
         </div>
         <div className="cities">
-          <div className="cities__places-container container">
-            <section className="cities__places places">
-              <h2 className="visually-hidden">Places</h2>
-              <b className="places__found">312 places to stay in Amsterdam</b>
-              <Sorting />
-              {<OffersList offers={offers} />}
-            </section>
-            <div className="cities__right-section">
-              <section className="cities__map map"></section>
+          {!cityOffers.length && <EmptyMainPage city={activeCity.name} />}
```

Comment:
Здесь как раз лучше подойдет тернарник
```suggestion
          {!cityOffers.length ? (
            <EmptyMainPage city={activeCity.name} />
          ) : (
            ........
          )}
```